.travis.yml

```diff
@@ -56,7 +56,6 @@ notifications:
           - http://julia.mit.edu:8000/travis-hook
 before_install:
     - make check-whitespace
-    - JULIA_SYSIMG_BUILD_FLAGS="--output-ji ../usr/lib/julia/sys.ji"
```

Comment:
> If this doesn't need to be deleted to add new functionality or fix the exact bug that this PR fixes,

You wouldn't let me delete it in the PR that really did need to remove it to add new functionality: https://github.com/JuliaLang/julia/pull/16059#discussion_r61190131. This whole business is really just outdated cruft now; at some point I'll work more on #5155 to make that more apparent and rewrite `build_sysimg.jl` too. I'm updating it here to make sure this test is actually covering the cases that I'm concerned this change might affect.